docs/getting-started/5-output-rails/README.md

```diff
@@ -160,15 +161,67 @@ As we can see, the LLM did generate the message containing the word "idiot", how
 
 The following figure depicts the process:
 
-<div align="center">
-<img src="../../_static/puml/output_rails_fig_1.png" width="815">
-</div>
+```{image} ../../_static/puml/output_rails_fig_1.png
+```
+
+## Streaming Output
+
+By default, the output from the rail is synchronous.
+You can enable streaming to provide asynchronous responses and reduce the time to the first response.
+
+1. Modify the `rails` field in the `config.yml` file and add the `streaming` field to enable streaming:
+
+   ```{code-block} yaml
+   :emphasize-lines: 9,10,12
+
+   rails:
+     input:
+       flows:
+         - self check input
+
+     output:
+       flows:
+         - self check output
+       streaming:
+          window_size: 20
```

Comment:
```suggestion
       streaming:
          chunk_size: 200
```

shall we drop it here and have it in `Streaming Output Configuration`

Reply:
I want a brief example here because I think it is more discoverable than on the config page.  But, it needs to link to the config page for the concepts and reference information.